package.json

```diff
@@ -1,68 +1,58 @@
 {
   "name": "geoguess",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "GeoGuess is an open-source geography game with Google Map StreetView.",
   "author": "Bilel Jegham and others",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "test:unit": "cross-env NODE_OPTIONS=--unhandled-rejections=warn vue-cli-service test:unit",
+    "serve": "vite serve",
+    "dev": "vite dev",
+    "build": "vite build",
+    "lint": "eslint .",
     "test:e2e": "cypress run",
-    "lint": "vue-cli-service lint",
     "cypress": "cypress",
-    "test": "npm run lint && npm run test:unit && npm run test:e2e",
-    "test:mutation": "cross-env NODE_OPTIONS=--unhandled-rejections=warn NODE_ENV=test BABEL_ENV=test VUE_CLI_TRANSPILE_BABEL_RUNTIME=true VUE_CLI_BABEL_TARGET_NODE=true VUE_CLI_BABEL_TRANSPILE_MODULES=true stryker run"
+    "test": "npm run lint && npm run test:e2e",
```

Comment:
```suggestion    
    "test": "npm run lint && npm run test:unit && npm run test:e2e",
```

Reply:
Vitest is a possibility for unit tests though it would take a lot of work to migrate